tests/test_pdo.py

```diff
@@ -155,20 +155,54 @@ def test_set_pdo_maps_to_slave_exception(motion_controller, rpdo_maps, tpdo_maps
         mc.capture.pdo.set_pdo_maps_to_slave(rx_maps, tx_maps, alias)
 
 
-@pytest.mark.parametrize(
-    "refresh_rate",
-    [
-        0.0001,
-        5,
-    ],
-)
 @pytest.mark.soem
-def test_pdos_refresh_rate(motion_controller, refresh_rate):
+def test_pdos_min_refresh_rate(motion_controller):
     mc, alias, environment = motion_controller
+    refresh_rate = 0.0001
     with pytest.raises(ValueError):
         mc.capture.pdo.start_pdos(COMMUNICATION_TYPE.Ethercat, refresh_rate)
 
 
+@pytest.mark.soem
+def test_pdos_watchdog_exception_auto(motion_controller):
+    exceptions = []
+
+    def exception_callback(exc):
+        exceptions.append(exc)
+
+    mc, alias, environment = motion_controller
+    refresh_rate = 3.5
+    mc.capture.pdo.subscribe_to_exceptions(exception_callback)
+    mc.capture.pdo.start_pdos(COMMUNICATION_TYPE.Ethercat, refresh_rate)
+    time.sleep(1)
+    mc.capture.pdo.unsubscribe_to_exceptions(exception_callback)
+    mc.capture.pdo.stop_pdos()
```

Comment:
I'm surprised you need to stop_pdos. Have they really started if the watchdog is too high?

Reply:
If the stop_pdos is not called the subsequent tests fail because the PDO Thread is detected as active.  https://github.com/ingeniamc/ingeniamotion/blob/0.9.0/ingeniamotion/pdo.py#L581

Reply:
Oh, on exit of the thread I would assume it notifies it's state to the model. Out of the scope of this issue. Created INGK-1056